client/src/js/components/bhRendererDropdown.js

```diff
@@ -8,35 +8,63 @@ angular.module('bhima.components')
     controller : bhRendererController
   });
 
-bhRendererController.$inject = [ 'appcache' ];
+bhRendererController.$inject = [ 'appcache', '$httpParamSerializer' ];
 
-function bhRendererController(AppCache) {
+function bhRendererController(AppCache, $httpParamSerializer) {
   var $ctrl = this;
 
   var cache = new AppCache('bhRendererComponent');
 
   // delay between GET request completion and loading indication, this is used
   // to compensate for the delay in browsers opening the print dialog
+  // @todo - make this work!
```

Comment:
Yeah ... it currently doesn't actually have any loading indicator.  I'm actually not sure how to get the loading status of a `a[download]` tag.  Do you know how to?